tests/flow/test_udf.py

```
@@ -0,0 +1,1035 @@
+        self.env.assertEqual(res["attributes"]["role"], "dev")
+
+    def test_load_invalid_invocations(self):
+        """
+        Test invalid invocations of GRAPH.UDF LOAD:
+        1. Missing arguments (should error).
+        2. Invalid trailing option (should error).
+        """
+
+        try:
+            self.db.execute_command("GRAPH.UDF", "LOAD")
+            assert False, "Expected failure on missing args"
```

Ruff (0.12.2)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)